src/iuse_actor.cpp

```diff
@@ -917,3 +917,109 @@ bool extended_firestarter_actor::can_use( const player* p, const item* it, bool
 
     return true;
 }
+
+iuse_actor *musical_instrument_actor::clone() const
+{
+    return new musical_instrument_actor(*this);
+}
+
+void musical_instrument_actor::load( JsonObject &obj )
+{
+    moves_cost = obj.get_int( "moves_cost", 25 );
+    volume = obj.get_int( "volume", 99 ); // Make it huge to alert the modder
```

Comment:
> Make it huge to alert the modder

I don't get this. If it's mandatory, why not omit the default value and let the json parser throw an error when it's missing?